packages/process/src/node/terminal-process.ts

```diff
@@ -17,164 +17,30 @@
 import { injectable, inject, named } from 'inversify';
 import { ILogger } from '@theia/core/lib/common';
 import { Process, ProcessType, ProcessOptions, ProcessErrorEvent } from './process';
-import { RawProcessOptions } from './raw-process';
 import { ProcessManager } from './process-manager';
 import { IPty, spawn } from '@theia/node-pty';
 import { MultiRingBuffer, MultiRingBufferReadableStream } from './multi-ring-buffer';
 import { DevNullStream } from './dev-null-stream';
 import { signame } from './utils';
 import { Writable } from 'stream';
 
-export type QuotingType = 'escaped' | 'strong' | 'weak';
```

Comment:
I think all these deletions should be documented at least in the changelog, since they will break anyone who references them.